src/main/js/models/helpers/getDisplayValue.js

```diff
@@ -124,33 +129,36 @@ const format = f.curryN(2)(
 
     const valueArray = (f.isArray(displayValue)) ? displayValue : [displayValue];
     // replace all occurences of {{n+1}} with displayValue[n]; then recur with n = n+1
+    // Because the formatPatterns consists of absolute columnId we first have to map index to columnId
     const applyFormat = function (result, dVal = valueArray, i = 1) {
       return (f.isEmpty(dVal))
         ? result
         : applyFormat(
-          result.replace(moustache(i), f.trim(f.first(dVal))),
+          result.replace(moustache(index2columnId(column)[i - 1]), f.trim(f.first(dVal))),
```

Comment:
Den Zugriff auf den index `[i - 1]` könnte man auch gleich in die `index2columnId` integrieren. Durch das o.g. `f.flow` und einem einfachen `f.get` ist das auch recht übersichtlich.

Reply:
Das ist anders wirklich sehr viel schöner.